src/NewCommand.php

```diff
@@ -243,4 +284,73 @@ protected function findComposer()
 
         return 'composer';
     }
+
+    /**
+     * @param string $packages
+     * @param string $composer
+     * @return string
+     */
+    private function getLaravelPackages(string $packages, string $composer)
+    {
+        $laravelPackages = explode(',', $packages);
+
+        $composerPackages = '';
+        foreach ($laravelPackages as $laravelPackage) {
+            if (array_key_exists($laravelPackage, self::LARAVEL_PACKAGES)) {
+                $composerPackages .= ' ' . self::LARAVEL_PACKAGES[$laravelPackage];
+            }
+        }
+
+        if ($composerPackages !== '') {
+            return $composer . ' require' . $composerPackages;
+        }
+
+        return $composerPackages;
+    }
+
+    /**
+     * @param string $packages
+     * @param string $composer
+     * @return string
+     */
+    private function getLaravelDevPackages(string $packages, string $composer)
+    {
+        $laravelPackages = explode(',', $packages);
+
+        $composerPackages = '';
+        foreach ($laravelPackages as $laravelPackage) {
+            if (array_key_exists($laravelPackage, self::LARAVEL_DEV_PACKAGES)) {
+                $composerPackages .= ' ' . self::LARAVEL_DEV_PACKAGES[$laravelPackage];
+            }
+        }
+
+        if ($composerPackages !== '') {
+            return $composer . ' require --dev' . $composerPackages;
+        }
+
+        return $composerPackages;
+    }
+
+    /**
+     * @param string $packages
```

Comment:
Please put 2 spaces before and after the word string in param docs, in all cases.

Reply:
I think it's ok now, right?